_posts/2023-07-17-video-library-launching.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Launching the Open Seeds Video Library"
+authors:
+- bebatut
+image: https://images.pexels.com/photos/114820/pexels-photo-114820.jpeg
+photos:
+  name: Mike Bird
+  url: https://www.pexels.com/photo/elvis-presley-digital-wallpaper-114820/
+  licence: CC-BY
+---
+
+{% assign cohort-calls = 0 %}
+{% for w in site.data.ols-1-schedule.weeks %}
```

Comment:
Hi @bebatut... 👋 
With the new structure of the data folder, this loop may need to be revisited.

Reply:
I know. I would probably merge this PR first as I need it by Tuesday and then update the redundancy PR to fix that

Reply:
Okay. 👍🏼